app/javascript/shared/assets/stylesheets/spacing.scss

```diff
@@ -8,6 +8,7 @@
   --space-one: 1rem;
   --space-slab: 1.2rem;
   --space-normal: 1.6rem;
+  --space-one-point-five: 1.5rem;
```

Comment:
We don't usually need this. Would be better to use as a local variable.

Reply:
Resolved